src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,40 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
```

Comment:
`health` is not configurable
```suggestion
  constructor(name) {
    this.name = name;
    this.health = 100;
```